README.md

```diff
@@ -100,6 +100,7 @@ Winter CMS is also compatible with most OctoberCMS plugins available at https://
 - [JaxWilko.Migrator](https://packagist.org/packages/jaxwilko/datamigrator) - A plugin that converts your database data to flat files.
 - [JaxWilko.MediaStream](https://packagist.org/packages/jaxwilko/wn-mediastream-plugin) - Adds streaming support for html5 videos.
 - [JaxWilko.OpcachePreload](https://packagist.org/packages/jaxwilko/wn-opcachepreload-plugin) - Winter CMS support for opcache preloading.
+- [LucaCalcaterra.LdapAuth](https://github.com/lucacalcaterra/wn-ldapauth-plugin) - LDAP Authentication for Winter CMS.
```

Comment:
Please put this in the Community section immediately below in the correct alphabetical order. Thanks!